### DISCUSSION TITLE
HTTP Framework for Time-Based Access to Resource States -- Memento

### DISCUSSION THREAD

**Discussion by ahmadnassri** [Kong/kong#7376] [Ideas and feature requests]
>  The HTTP-based Memento framework bridges the present and past Web.
>    It facilitates obtaining representations of prior states of a given
>    resource by introducing datetime negotiation and TimeMaps.  Datetime
>    negotiation is a variation on content negotiation that leverages the
>    given resource's URI and a user agent's preferred datetime.  TimeMaps
>    are lists that enumerate URIs of resources that encapsulate prior
>    states of the given resource.  The framework also facilitates
>    recognizing a resource that encapsulates a frozen prior state of
>    another resource.

-- [RFC 7089](https://tools.ietf.org/html/rfc7089)

**Comment by thibaultcha** [comment]
Same. just some spec, no description, what is this supposed to mean?

**Comment by ahmadnassri** [comment]
feature request to implement the spec

**Comment by shivaspk** [comment]
Spec Link: https://mementoweb.org/guide/rfc/#rfc.section.1
